### PR TITLE
Feature/more apis

### PR DESCRIPTION
**Pull request by provokateurin**
Depends on https://github.com/nextcloud/openapi-extractor/issues/8 as I had to patch the files and core specs.

### CODE REVIEW

File: external/nextcloud-openapi-extractor
Comment (Leptopoda) [inline external/nextcloud-openapi-extractor:1]: Why did you remove the extractor?

Comment (provokateurin) [reply external/nextcloud-openapi-extractor:1]: The extractor is now a dev dependency of the server. That way we can be sure the server code is compatible with the used extractor version. The same will apply for apps as they will also have their own extractor dependency pinned.

**Comment by provokateurin** [comment]
Will be ready once https://github.com/nextcloud/neon/pull/597 is merged

**Review by Leptopoda — APPROVED** [review]
